src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -83,6 +89,9 @@ internal MvcRazorHost(ICodeTreeCache codeTreeCache, RazorPathNormalizer pathNorm
 
                     WriteTagHelperAsyncMethodName = "WriteTagHelperAsync",
                     WriteTagHelperToAsyncMethodName = "WriteTagHelperToAsync",
+
+                    // IHtmlHelper<TModel> is also not accessible here.
```

Comment:
`Can't use nameof because IHtmlHelper<TModel>.Raw is not accessible here.` - Had to read the comment over several times and look at previous configurations to "sort of" understand what it meant.